core/storage/user/gae_models.py

```diff
@@ -176,6 +176,16 @@ class ExpUserLastPlaythroughModel(base_models.BaseModel):
 
     @classmethod
     def _generate_id(cls, user_id, exploration_id):
+        """Generates key for the instance of ExpUserLastPlaythroughModel
+        class in the required format with the arguments provided.
+
+        Args:
+            user_id: str. The id of the user.
+            exploration_id: str. The id of the exploration.
+        Returns:
```

Comment:
This doesn't seem to be done. Add blank line before Returns

Reply:
Done! Sorry for missing it.